compiler/src/main/java/com/squareup/anvil/compiler/codegen/dagger/MembersInjectionUtils.kt

```diff
@@ -0,0 +1,56 @@
+package com.squareup.anvil.compiler.codegen.dagger
+
+import com.squareup.anvil.compiler.codegen.Parameter
+import com.squareup.anvil.compiler.daggerDoubleCheckFqNameString
+import com.squareup.anvil.compiler.injectFqName
+import com.squareup.anvil.compiler.internal.capitalize
+import com.squareup.anvil.compiler.internal.generateClassName
+import com.squareup.anvil.compiler.internal.hasAnnotation
+import com.squareup.kotlinpoet.ClassName
+import com.squareup.kotlinpoet.FunSpec
+import org.jetbrains.kotlin.descriptors.ModuleDescriptor
+import org.jetbrains.kotlin.lexer.KtTokens
+import org.jetbrains.kotlin.psi.KtClassBody
+import org.jetbrains.kotlin.psi.KtClassOrObject
+import org.jetbrains.kotlin.psi.KtProperty
+import org.jetbrains.kotlin.psi.psiUtil.visibilityModifierTypeOrDefault
+
+// TODO
+//  Include methods: https://github.com/square/anvil/issues/339
+//  Include superclass: https://github.com/square/anvil/issues/343
+internal fun KtClassOrObject.injectedMembers(module: ModuleDescriptor) = children
+  .asSequence()
+  .filterIsInstance<KtClassBody>()
```

Comment:
Does this work with nested classes?

Reply:
I can add a test. The implementation was the same in all three places so if it worked in one it shouldn't be different here

Reply:
Test in 31c01cc